### PR TITLE
Making setup_efm to support Debian and also added force_initdb

### PR DESCRIPTION
**Pull request by vibhorkum**
With this PR following are the changes:
1. Support Debian in setup_efm
2. force_initdb changes for setup_replication
3. forec_initdb changes init_dbserver

### CODE REVIEW

File: roles/setup_efm/tasks/prepare_hba_value_list.yml [hunk 1]
```diff
@@ -1,21 +1,46 @@
...
+      ]
   when: not pg_ssl
+  loop: "{{ pg_cluster_nodes }}"
+  loop_control:
+    loop_var: node
+  run_once: true
+  no_log: "{{ disable_logging }}"
 
-- name: Prepare the list
+- name: Prepare hba non ssl list
```
Comment (jt-edb) [inline roles/setup_efm/tasks/prepare_hba_value_list.yml:26]: I guess this is the "hba ssl" list, isn't it?

Comment (jt-edb) [inline roles/setup_efm/tasks/prepare_hba_value_list.yml:26]: I guess this the "HBA SSL" list, isn't it?

**Review by jt-edb — COMMENTED** [review]
LGTM! Only one minor comment. Thank you

File: roles/setup_efm/tasks/prepare_hba_value_list.yml (hunk 1 above)
Comment (vibhorkum) [reply roles/setup_efm/tasks/prepare_hba_value_list.yml]: fixed

**Review by jt-edb — APPROVED** [review]
LGTM

**Pull request merged**